### ISSUE TITLE
[MNG-7501] display relative path to pom.xml

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#8956]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** opened **[MNG-7501](https://issues.apache.org/jira/browse/MNG-7501?redirect=false)** and commented

when building large multi-module project, when a failure happens in the middle of the build, it's not easy to even identify where the module is located in the source tree: Maven displays the module name, but not the path to pom.xml. Then often we have to read output log of goals that have run hoping to find a hint

it would be nice to have by default the path to the pom.xml displayed during Maven run.

I see 2 options:
1. either in the module build header:

```
[INFO] --< org.apache.maven.its.plugins:maven-it-plugin-dependency-resolution >--
[INFO] Building Maven IT Plugin :: Dependency Resolution 2.1-SNAPSHOT   [31/78]
[INFO] core-it-support/core-it-plugins/maven-it-plugin-dependency-resolution/pom.xml
[INFO] ----------------------------[ maven-plugin ]----------------------------
```

2. or in each goal execution line during the module build:

```
[INFO] --- maven-enforcer-plugin:1.4.1:enforce (enforce-maven-version) @ maven-it-plugin-dependency-resolution core-it-support/core-it-plugins/maven-it-plugin-dependency-resolution/pom.xml ---
```

---

**Affects:** 3.8.6

**Remote Links:**
- [GitHub Pull Request #756
](https://github.com/apache/maven/pull/756)

**Backported to:** [4.0.0-alpha-2](https://github.com/apache/maven/milestone/113?closed=1), [3.9.0](https://github.com/apache/maven/milestone/2?closed=1), [3.8.8](https://github.com/apache/maven/milestone/109?closed=1)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Will this be included in 4.0.0 also?

**Comment by jira-importer** [comment]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** commented

yes, sure, it will be merged to 4.0.0 also

**Comment by jira-importer** [comment]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** commented

done:
- in https://github.com/apache/maven/commit/c9a3e0e45b0a6e7eca29d150443b04c9c99f3813 for Maven 3.9.x
- in https://github.com/apache/maven/commit/4e88d62ab23a2cef8b1ec8af479353bc4d86fe25 for master / Maven 4

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [47bf22022d79a611cdbecc49cc2fa51d84306835](https://gitbox.apache.org/repos/asf?p=maven.git&a=commit&h=64da2c58243bc6f48827301b744b9f6312497550) for `maven-3.8.x` branch.
